ci-infra/docker-image/build.sh

```diff
@@ -27,7 +27,7 @@ REPO_DIR="$(realpath "$THIS_SCRIPT_DIR/../.." )"
 cat "$REPO_DIR"/{docs,test,sanity,mypy}.requirements > all.requirements
 # TODO but is ghrc.io feature available/enabled?
 # Or push to quay.io?
-docker build "$DOCKER_BUILD_OPT" -t "$DOCKER_REGISTRY_REPO:$DOCKER_IMAGE_TAG" .
+docker build $DOCKER_BUILD_OPT -t "$DOCKER_REGISTRY_REPO:$DOCKER_IMAGE_TAG" .
```

Comment:
Was removing the quotes here intentional? I usually keep variables quoted when I use them as command arguments to avoid word splitting, but it looks like that won't matter here. Just checking.

Reply:
Removing quotes was required. A bit older docker-ce client was happy with empty DOCKER_BUILD_OPT - it just ignored "". But most recent one did complain.